storm-core/src/clj/org/apache/storm/daemon/supervisor.clj

```diff
@@ -344,6 +348,12 @@
    :sync-retry (atom 0)
    :download-lock (Object.)
    :stormid->profiler-actions (atom {})
+   :cgroup-manager (if (conf STORM-CGROUP-ENABLE)
+                     (let [cgroup-manager (.newInstance (Class/forName (conf STORM-RESOURCE-ISOLATION-PLUGIN)))]
```

Comment:
If the resource isolation plugin is pluggable, do we want to call this `:cgroup-manager`?